### PR TITLE
AI doc update - Tile

### LINKED ISSUE #4040: AI documentation: Add AI docs for Tile
Update the checkbox component page using the [AI component guidelines template](https://github.com/carbon-design-system/carbon-website/issues/4007).

## Usage tab
[See example in Figma](https://www.figma.com/file/OJwBWBlctEgm9zYUCYbhu3/Carbon-for-AI-%E2%80%93-website-integration?type=design&node-id=80%3A7220&mode=design&t=XPoeQ0K8Ea0VbhbW-1)

- [x] Add inline notification to the top of the page under intro paragraph. The link should go to the AI component’s storybook example.
     - There is already an exisiting inline notification for feature flag, `figure out different ways to accommodated either both or maybe 1`
- [x] Add “AI presence” to the anchor links.
- [x] Add an H2 section for “AI presence” between “Modifiers” (or “Behaviors”) and “Related” content sections.
- [x] Copy the content from the template example and edit the template content to be specific to each component. Only the input components will get the extra revert h3 sections.
- [x] Create images for the components similar to the template examples (add images to the [Usage and Style images](https://www.figma.com/file/9J3QEJ717fLUa7tzlWVMMu/Usage-and-Style-images?type=design&t=VpbAnlbxwfthmkMr-6) file)

_If links to the general AI guidelines sections aren’t ready yet then don’t include links or hyperlinks yet. We will add these in later when ready._

## Style
[See example in Figma](https://www.figma.com/file/OJwBWBlctEgm9zYUCYbhu3/Carbon-for-AI-%E2%80%93-website-integration?type=design&node-id=80%3A7192&mode=design&t=XPoeQ0K8Ea0VbhbW-1)

- [x] Add “AI presence” to the anchor links.
- [x] Add an H2 section for “AI presence” at the end of the page
- [x] Copy the content from the template example and edit the template content where needed to be specific to each component.
- [x] Create an image for the component similar to the template examples. (add images to the [Usage and Style images](https://www.figma.com/file/9J3QEJ717fLUa7tzlWVMMu/Usage-and-Style-images?type=design&t=VpbAnlbxwfthmkMr-6) file)

_If links to the general AI guidelines sections aren’t ready yet then don’t include links or hyperlinks yet. We will add these in later when ready._

## Accessibility 
No changes needed at the moment.

---

# Deliverables
- [x] Pull request opened in the website repo with content updates

### PR DESCRIPTION
**Pull request by Kritvi-bhatia17**
Closes #4040 

### Added in the Usage tab
- Inline notification
- AI presence section: text with image

### Added in the Style tab
- AI presence section: text specs with image 

_Bug fixes:_
- Page description
- Anchor links
- Feedback section

### CODE REVIEW

File: src/pages/components/tile/usage.mdx [hunk 1]
```diff
@@ -498,6 +505,25 @@ what content it contains but all types of tiles can take focus.
...
+
+<Row>
+<Column colLg={8}>
+
+![Tile with AI presence](images/tile-AI-presence-usage.png)
+
+</Column>
+</Row>
+
+<Caption>Tile with AI presence</Caption>
```
Comment (aagonzales) [inline src/pages/components/tile/usage.mdx:525]: I don't think you need this caption but if you're going to have it I would be more specific with it so it more accurately describes the image.
```suggestion
<Caption>Base, clickable, single-select and multi-select tiles with AI presence</Caption>
```

File: src/pages/components/tile/style.mdx [hunk 2]
```diff
@@ -9,6 +9,40 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
...
+
+<AnchorLink>Color</AnchorLink>
+<AnchorLink>Typography</AnchorLink>
+<AnchorLink>Structure</AnchorLink>
+<AnchorLink>AI presence</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
+<PageDescription>
```
Comment (aagonzales) [inline src/pages/components/tile/style.mdx:29]: Page description and anchor links are repeated here. Remove the duplications.

**Review by aagonzales — COMMENTED** [review]
Just some small things.

**Review by alisonjoseph — APPROVED** [review]
This looks good (after Anna's small issues are fixed)

File: src/pages/components/tile/usage.mdx (hunk 1 above)
Comment (Kritvi-bhatia17) [reply src/pages/components/tile/usage.mdx:525]: Oh I thought it was a single image, so thought of adding the caption. 
Will remove it in that case

File: src/pages/components/tile/style.mdx (hunk 2 above)
Comment (Kritvi-bhatia17) [reply src/pages/components/tile/style.mdx:29]: Ohh damn, idk how it happended lol but good catch thanks!

**Review by aagonzales — APPROVED** [review]
Looks good!

**Review by guidari — APPROVED** [review]

**Pull request merged**